sites/footers/_base.scss

```diff
@@ -56,7 +148,7 @@
 	}
 
 	.tofpg {
```

Comment:
Did we removed it? If this is for backward compatibility, please add a comment about it.